brownie-config.yaml

```diff
@@ -0,0 +1,66 @@
+project_structure:
+    build: build
```

Comment:
Could we put everything vyper-related into a vyper root folder?

Reply:
ofc, including the tests and contracts, or just the brownie-related files?

Reply:
Let’s do it all

Reply:
like this?

```
project_structure:
    build: vyper/build
    contracts: vyper/contracts/
    interfaces: vyper/interfaces
    reports: vyper/reports
    scripts: vyper/scripts
    tests: vyper/test
```

Reply:
Exactly

Reply:
done